### DISCUSSION TITLE
配置MySQL JDBC Catalog时include_database_list参数无效

### DISCUSSION THREAD

**Discussion by Once2012** [apache/doris#21289] [A - General / Q&A]
版本说明：
Doris: 1.2.4.1
OS: Ubuntu 20.04
JDK: openjdk-11
安装: 本机1FE + 1BE
MySQL: 5.7
MySQL Driver: 8.0.27
MySQL Driver Class: com.mysql.cj.jdbc.Driver

当我配置MySQL JDBC Catalog时，采用如下SQL语句：
CREATE CATALOG jdbc_mysql_35 PROPERTIES (
    "type"="jdbc",
    "user"="user1",
    "password"="xxxxx",
    "jdbc_url" = "jdbc:mysql://xxxx:3306/testdb1?useUnicode=true&characterEncoding=utf8&zeroDateTimeBehavior=convertToNull&useSSL=true&serverTimezone=GMT%2B8",
    "driver_url" = "mysql-connector-java-8.0.27.jar",
    "driver_class" = "com.mysql.cj.jdbc.Driver",
    "only_specified_database" = "true",
    "include_database_list" = "testdb1,testdb2"
);

但配置好之后，testdb1的表能访问，但testdb2的表无法访问，报如下错：
1049 - errCode = 2, detailMessage = Unknown database 'default_cluster:testdb2'

注：user1账号拥有所有两个库的访问权限。

曾做过的尝试：
1）将驱动类从com.mysql.cj.jdbc.Driver改为com.mysql.jdbc.Driver，无效
2）将驱动从mysql-connector-java-8.0.27.jar改为mysql-connector-java-5.1.49.jar，无效

麻烦哪位高人帮忙看看，谢谢啦

**Comment by zy-kkk** [comment]
Version 1.2 is no longer maintained. The latest version 2.0 release or 2.1 release does not have this problem. Please upgrade and use
